QRCode/examples/Example.sol

```diff
@@ -13,11 +14,16 @@ contract Example is Debot {
 
     function setResult(string value) public {
         // TODO: continue here, validate value and so on...
+        Terminal.print(0, format("{}", value));
```

Comment:
What do you think of using prompt here? (as well as for `get` method)?

Reply:
prompt for `scan` is a good idea!

Reply:
But it should be optional. I can give you an example: i have a menu which has a button with description what to scan and i don't want to print additional `prompt`.